bucket/kicad-lite-nightly.json

```diff
@@ -0,0 +1,39 @@
+{
+    "version": "r27528.623dc8980f.2022-05-09",
+    "description": "Electronics Design Automation Suite, nightly build without libraries",
+    "homepage": "https://kicad.org",
+    "license": "GPL-3.0-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://kicad-downloads.s3.cern.ch/windows/nightly/kicad-nightly-r27528.623dc8980f-x86_64-lite.exe#/dl.7z",
+            "hash": "03b6db02695f9f759878555adc1b298b4f543b050386afe249359ac676a7ba55"
+        }
+    },
+    "pre_install": "Remove-Item \"$dir\\`$*\", \"$dir\\Uninst*\" -Recurse",
+    "bin": [
+        [
+            "bin\\kicad.exe",
```

Comment:
CLI args?

Reply:
You can load a project file from the command line.

Reply:
In that case, kicad-nightly is fine, so that people can install stable and nightly side by side

Reply:
Sure thing, done.